.azure-pipelines/azure-pipelines-build.yml

```diff
@@ -114,12 +114,19 @@ jobs:
             docker_syncd_rpc_image: yes
             platform_rpc: nephos
 
+        - name: pensando
+          pool: sonicbld-arm64
+          variables:
+            PLATFORM_ARCH: arm64
+
     buildSteps:
       - template: .azure-pipelines/template-skipvstest.yml@buildimage
       - template: .azure-pipelines/template-daemon.yml@buildimage
       - bash: |
           set -ex
-          if [ $(GROUP_NAME) == vs ]; then
+          if [ $(GROUP_NAME) == pensando ]; then
+            make $BUILD_OPTIONS target/sonic-pensando.tar
```

Comment:
pensando does not have the image .bin or image.gz?

Reply:
In makefile rules, they only have .tar image.
When building image, they have .bin target file.
In previous onboarding pensando platform PR. They added a device type dsc. So, I didn't comment on image extensions.